doc/design/requirements.md

```diff
@@ -0,0 +1,69 @@
+# Requirements and goals
+
+Includes the overlapping projects of TiUp, tiup-cluster, and tiup-playground.
+
+We should distinguish between goals and requirements for the long term vs an initial release.
+
+## Vision and high-level goals
+
+* Manage and deploy TiDB and TiKV clusters
```

Comment:
* Expand the TiDB ecology (including but not limited to manage and deploy TiDB clusters)

Reply:
Sorry, I don't understand "Expand the TiDB ecology"

Reply:
I hope we can build a bigger TiDB ecology base on the TiUP because we allow the community to upload components developed by themself, and we think some ideas we cannot think of, but someone in the community can.